src/inc/utilcode.h

```diff
@@ -5110,7 +5110,7 @@ BOOL IsIPInModule(HMODULE_TGT hModule, PCODE ip);
 struct CoreClrCallbacks
 {
     typedef IExecutionEngine* (__stdcall * pfnIEE_t)();
-    typedef HRESULT (__stdcall * pfnGetCORSystemDirectory_t)(SString& pbuffer);
+    typedef HRESULT (STDAPICALLTYPE * pfnGetCORSystemDirectory_t)(SString& pbuffer);
```

Comment:
It seems to me that all of the callbacks should be of STDAPICALLTYPE, as well as the EEHeapAllocInProcessHeap / EEHeapFreeInProcessHeap functions. @jkotas what do you think?